### DISCUSSION TITLE
Trying to display an image on frontend but for some reason it's 1px by 1px

### DISCUSSION THREAD

**Discussion by twan-2022** [area17/twill#1745] [Q&A]
I am trying to display an image I added in my Twill cms but for some reason it keeps displaying a base64 string that shows a 1 by 1 px image. 

In my product model I've got these mediaparams:

```
public $mediasParams = [
    'cover' => [
        'default' => [
            [
                'name' => 'default',
                'ratio' => 16 / 9,
            ],
        ],
        'mobile' => [
            [
                'name' => 'mobile',
                'ratio' => 1,
            ],
        ],
        'flexible' => [
            [
                'name' => 'free',
                'ratio' => 0,
            ],
            [
                'name' => 'landscape',
                'ratio' => 16 / 9,
            ],
            [
                'name' => 'portrait',
                'ratio' => 3 / 5,
            ],
        ],
    ],
];
```
I then try to display it like this:

`<img src="{{ $product->image('cover') }}" alt="{{ $product->title }}">`

But this returns:

`data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7`

Which is a white pixel image of 1 by 1.

I've also tried:  

`<img src="{{ $product->image('cover', 'default') }}" alt="{{ $product->title }}">`

But this has the same result. How am I supposed to display media?

**Comment by haringsrob** [comment]
Hey @twan-2022,

In the Twill user interface, do you see the correct crops?

**Reply by twan-2022** [reply]
> Hey @twan-2022,
> 
> In the Twill user interface, do you see the correct crops?

This is what I see for the image I attached to a product:

001.jpg
Original: 600 × 600
Desktop crop: 600 × 338

Tablet crop: 600 × 450

Mobile crop: 600 × 600

**Reply by haringsrob** [reply]
Could you consult the database and see what is in the mediables rows for this product?

**Reply by twan-2022** [reply]
> Could you consult the database and see what is in the mediables rows for this product?

Yeah the bottom 3 rows are related:

![image](https://user-images.githubusercontent.com/11909652/179722385-6062a260-c910-48b4-9d3c-30a1fc1f6e6c.png)

**Reply by haringsrob** [reply]
Hmm so from what I can see there's no default crop.

Can you try `$product->image('image', 'desktop')`

**Reply by twan-2022** [reply]
> Hmm so from what I can see there's no default crop.
> 
> Can you try `$product->image('image', 'desktop')`

Yeah this works thanks! In the documentation I saw this line: `A role is a way to define different contexts in which a image might be placed. For example, roles for a People model could be profile and cover. ` And as I saw 'cover' in my created model I assumed this was what I had to add as role but I understand now it has to be 'image'

**Reply by haringsrob** [reply]
Great that it worked!
